addons/stock/models/product.py

```diff
@@ -1024,7 +1024,8 @@ class ProductCategory(models.Model):
              "LIFO: products/lots that were stocked last will be moved out first.\n"
              "Closet location: products/lots closest to the target location will be moved out first.\n"
              "FEFO: products/lots with the closest removal date will be moved out first "
-             "(the availability of this method depends on the \"Expiration Dates\" setting)."
+             "(the availability of this method depends on the \"Expiration Dates\" setting).\n"
+             "Least Packages: FIFO but allows to visit the less packages possible in the case there are several packages containing the same product"
```

Comment:
English cleanup: 

"FIFO but with the least number of packages possible in the case there are several packages containing the same product."